bin/compile

```diff
@@ -57,7 +57,7 @@ function indent() {
 # Hardcoded available stable versions
 # Use it to manually update the version as per https://firefox-source-docs.mozilla.org/testing/geckodriver/Support.html
 VERSION_FIREFOX=82.0.3
-VERSION_GECKODRIVER=0.28.0
+VERSION_GECKODRIVER=0.30.0
```

Comment:
Could we directly switch to version 0.31.0? https://github.com/mozilla/geckodriver/releases

Reply:
Geckodriver version 0.28.0 wasn't working because of `--websocket-port` option which was added in version 0.30.0 and I have tested the working of same version which works totally fine.
So we can use 0.31.0 as it's latest release, but I haven't tested it yet. Should we test it?
If it works fine then we can directly upgrade to 0.31.0. And if we're upgrading geckodriver then why not upgrade Firefox too!

Reply:
Yes, I'm all for testing 0.31.0, and I'm all for upgrading Firefox as well!